amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -225,11 +193,35 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
 
 
   // TODO: Hard-coded text strings should be translatable/customizable
-  getTabContent = (results) => {
-    const startIndex = (RESULTS_PER_PAGE * results.page_index) + 1;
-    const endIndex = RESULTS_PER_PAGE * ( results.page_index + 1);
-    let title =`${startIndex}-${Math.min(endIndex, results.total_results)} of ${results.total_results} results`;
+  getTabContent = (results, tabLabel) => {
+    const { searchTerm } = this.props;
+    const { page_index, total_results } = results;
+    const startIndex = (RESULTS_PER_PAGE * page_index) + 1;
+    const endIndex = RESULTS_PER_PAGE * ( page_index + 1);
+
+    // Check no results
+    if (total_results === 0 && searchTerm.length > 0) {
```

Comment:
nit: I'd be a fan of putting support into the Tab component itself for an `emptySectionMessage`. That way we can have consistent styling as the app scales and we potentially use this concept on other pages.

Reply:
Will leave a TODO and file a ticket.